ambry-cloud/src/main/java/com/github/ambry/cloud/azure/ADAuthBasedStorageClient.java

```diff
@@ -100,6 +103,18 @@ protected BlobServiceAsyncClient buildBlobServiceAsyncClient(HttpClient httpClie
 
   @Override
   protected void validateABSAuthConfigs(AzureCloudConfig azureCloudConfig) {
+    if (storageAccountInfo() != null) {
+      if (StringUtils.isNullOrEmpty(storageAccountInfo().getStorageScope())) {
```

Comment:
nit: Just to note, we also have a utility [method](https://github.com/linkedin/ambry/blob/master/ambry-utils/src/main/java/com/github/ambry/utils/Utils.java#L1282) for checking if string is null or empty.

Reply:
Noted for future